PlayCover/zh-Hans.lproj/Localizable.strings

```diff
@@ -73,18 +73,18 @@
 "App cache was cleared!" = "已清空应用缓存！";
 "Keymapping imported!" = "已导入键盘映射！";
 "Error during import" = "导入时发生错误";
-"With this you can bind touch layout to bind touch controls to mouse and keyboard. Use Control + P to open editor menu." = "With this you can bind touch layout to bind touch controls to mouse and keyboard. Use Control + P to open editor menu.";
-"Use this feature to play games in fullscreen and adapt app window to another dimensions." = "Use this feature to play games in fullscreen and adapt app window to another dimensions.";
+"With this you can bind touch layout to bind touch controls to mouse and keyboard. Use Control + P to open editor menu." = "借此你可以将触控按键布局和触控控制绑定到鼠标和键盘上。点击Control + P打开编辑菜单。";
+"Use this feature to play games in fullscreen and adapt app window to another dimensions." = "使用本功能以全屏进行游戏或将App窗口适应其他分辨率。";
 "Enable Jailbreak Bypass (Alpha)" = "绕过越狱检测（测试版）";
-"ATTENTION: Don't enable for some games yet! This allows you to bypass bans that don't let you login" = "ATTENTION: Don't enable for some games yet! This allows you to bypass bans that don't let you login";
+"ATTENTION: Don't enable for some games yet! This allows you to bypass bans that don't let you login" = "注意：对于某些游戏不要启用！这允许你绕过某些使你无法登录的限制。";
 "Screen refresh rate" = "屏幕刷新率";
 "Mouse sensivity: %.f" = "鼠标灵敏度：%.f";
 "Reset settings and keymapping" = "恢复默认设置";
 "Settings reseted to default!" = "已重置设置到默认值！";
 "Ok" = "OK";
-"No device with sample rate of 48 / 44.1 ghz found! Please, lower sample rate in Audio settings or connect another output audio device. Otherwise crashes possible!" = "No device with sample rate of 48 / 44.1 ghz found! Please, lower sample rate in Audio settings or connect another output audio device. Otherwise crashes possible!";
+"No device with sample rate of 48 / 44.1 ghz found! Please, lower sample rate in Audio settings or connect another output audio device. Otherwise crashes possible!" = "未找到采样率为48或44.1GHz的设备！请在音频设置中降低采样率或者连接其他的输出音频设备。否则可能会发生崩溃！";
```

Comment:
Mind the kerning and the usual comma usage in Chinese.
```suggestion
"No device with sample rate of 48 / 44.1 ghz found! Please, lower sample rate in Audio settings or connect another output audio device. Otherwise crashes possible!" = "未找到采样率为 48 或 44.1GHz 的设备！请在音频设置中降低采样率或者连接其他的输出音频设备，否则可能会发生崩溃！";
```

Reply:
Here there's no need to add a space in Chinese characters and Arabic numerals. You can see that there's no space between characters and numerals in the [articles on the website of the State Council of China](http://www.gov.cn/premier/2022-07/13/content_5700841.htm). Otherwise it looks really strange.

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/48173350/178921758-67c22fa5-ce69-4d9e-86ca-57db0d76a794.png">

Reply:
OK, thanks for your explanation, I will make a commit to resolve it and thanks for reviewing it again.